### ISSUE TITLE
CLI for Piwigo

### ISSUE THREAD

**Issue by hukoeth** [Piwigo/Piwigo#648]
It would be great if piwigo offered a script that can be used in a cronjob (i.e. CLI for Piwigo). This would e.g. be helpful for the e-mail notifications about new updates. I use Piwigo for my family only so the site is not visited very frequently.

Another use case could be to automatically sync albums and do some other administrative tasks.

**Comment by plegall** [comment]
A bit like tools/piwigo_remote.pl ? (which is more designed to be a testing tool for Piwigo API, but it's a good start)

**Comment by hukoeth** [comment]
Yes that looks indeed promising. I wasn't aware that this existed. I'll take a look at it.

**Comment by plegall** [comment]
I was wrong when I compared it to `piwigo_remote.pl`. The CLI should not rely on Piwigo API (thus on remote HTTP requests). That's the whole point of the CLI: do not depend on HTTP. I agree, more and more, that Piwigo should have a CLI tool: install, update, plugin install, synchronization...
